zerver/management/commands/change_auth_backends.py

```diff
@@ -0,0 +1,58 @@
+from argparse import ArgumentParser
+from typing import Any
+
+from typing_extensions import override
+
+from zerver.actions.realm_settings import do_set_realm_authentication_methods
+from zerver.lib.management import ZulipBaseCommand
+
+
+class Command(ZulipBaseCommand):
+    help = """Enable or disable an authentication backend for a realm"""
+
+    @override
+    def add_arguments(self, parser: ArgumentParser) -> None:
+        self.add_realm_args(parser, required=True)
+        group = parser.add_mutually_exclusive_group(required=True)
+        group.add_argument(
+            "--enable",
+            type=str,
+            help="Name of the authentication backend to enable",
+        )
+        group.add_argument(
+            "--disable",
+            type=str,
+            help="Name of the authentication backend to disable",
+        )
+        group.add_argument(
+            "--show",
+            action="store_true",
+            help="Show current authentication backends for the realm",
+        )
+
+    @override
+    def handle(self, *args: Any, **options: str) -> None:
+        realm = self.get_realm(options)
+        assert realm is not None  # Should be ensured by parser
+
+        if options["show"]:
+            print("Current authentication backends for the realm:")
+            print(realm.authentication_methods_dict())
+            return
+
+        auth_methods = realm.authentication_methods_dict()
+
+        if options["enable"]:
+            backend_name = options["enable"]
+            auth_methods[backend_name] = True
+            print(f"Enabling {backend_name} backend for realm {realm.name}")
+        elif options["disable"]:
+            backend_name = options["disable"]
+            auth_methods[backend_name] = False
+            print(f"Disabling {backend_name} backend for realm {realm.name}")
+
+        do_set_realm_authentication_methods(realm, auth_methods, acting_user=None)
+
+        print("Updated authentication backends for the realm:")
+        print(realm.authentication_methods_dict())
```

Comment:
I think it'd be significantly more readable to take a minute to list these out for human consumption -- e.g.:
```
Enabled backends:
  Dev
  Email
  GitLab
  Google
  Apple
  SAML
  OpenID Connect

Disabled backends:
  Github

Reply:
Yeah good idea, fixed and updated the commit message + PR description with the new output